plugins/@grouparoo/pipedrive/src/lib/export/persons/destinationMappingOptions.ts

```diff
@@ -5,9 +5,10 @@ import {
   objectCache,
 } from "@grouparoo/core";
 
-import { connect } from "../connect";
```

Comment:
let's move this to the same structure as we did with the tests together
* `export-persons`
* `export-organizations`